examples/python/tensor_transforms.ipynb

```diff
@@ -0,0 +1,611 @@
JTMw2SgUaDC0msgp3I5hKa8KI8RYlyVwKTRl7ch7hfUFupYfhN3PiXQIf5ayqqzSrK2RecijFsZtf330WpNM7/vMPN7pwlqPltd0BY4LUPBf3XbMA+vLScsX+niuzmDfo9EpbzmDdN4Djiq9Kg0jBKoRRWrmlYKjUWStFHa8Pa5dojvDimKHJsU3/eJ4wQSl8Bt4rouWWqKdcQCyLIc2zaCnWcZ62vr4DhEzSZ5PcQLfXIUbhBgeXVa9Sah4zPdnEIf0lx59lnwDkChIBsDW2CVWGtdAkk6nRImPYbBNlzOwckh63C7bcP+yT/4X/nrqwWjl2/geOPIU/PQfAuVaHw3JXADAr/ADi3IPOpti3hcEEYQ+AqnZWPlmmEnxvU1jm9e1sQdwbCPZbt4zTbt6RlQAXmWY1sOtq1Z3VjjiWefo9a+i8hrEK9vsnL5GazTz990ineGEhCYrVhWEgCMqFZioevewgjspKUwWZQTUQme+NWSNRDc/mTNgMR/GlR1AyFGIMXSkCo/OY49cQ4JTqbsBI9vqFz0Js5pleeTv0dUbolAzmsT5xWrRaDGZSxDA4GKsJgH1UT5BZ6/wlyYMF/rM1vL2d0AtxYTWGO2N5dYXrtG0PLon+xRFB7T0y3S2DHNeYD19YyLL1wnT3Nacw28Wp2ar1mYsklQO9PtlfUadQsiVcZ0NYSholYzGrAoNGM7w7U1lmdh2QUohU4Vtl8n8gPCKDQkH1pTq9dxbJs4TrBtm6gWMRgMGMRr5K6D26zRthVBEKK1Sz0Yo7yARrOBZ7tEU9O0vIjNa1cZHDlhGoHECWrjAs19++lcLEkWowg6Y8jKHHAxgKQHiYTpb08JvOdrjvAj/9ev0evdpD5g9xG4XqIHx2PodA0uAoXtaByrwHU1mc5ptANCe5Zhf0joG5/fsjX5KKXfT6CfULgDklRh2Q7KckBZBHGG4/qgHYphTKJzA+Pe6vLc4LP4U09w4uhx/EQRrI046tS5wkvSR8CdogRkVZYoekJlmovwiLneoRI+wfhLsE9W1ZL4dcfHb2GESIZYcZL/lxW9hjHj2y869gYVwk+CeMKEJMxBQiQiAtyhQgbOTMxX3BCJaUggU1yYAKOA7ir3WS+3WTPnsTw4srfJ/l3HgEPmnypFBR5u02V2LuTQoZiu6+JkLpvjdS6efpKt7Q7Xrh5gMBize89eTjz0IFpPYVk2tpPz+SdPkQ1i9u3dy7X1FaZnpnno2Dy5q3YAjf3SIos8qKmqvMKiBP/5iiyz0dohSxXK8g0FmANJEqN9G9+rEwQBQRCUrb00QeBjWRaBn+1UxI36CRQ2YW0KP2yaAiDPo8gVI3eM5do06k3TdKNWY+D0mJtbwMaCPCdPEuKNBQ49dIKnLp1G7bsXr9UmXtiCwRCuNwzr6mCzvLpJH/JWwyVGcf6vf5MsvonSGG5Xn3UB3T4kNjqqkSQ5ROC4FmHUolV3KVwP17WxlYVjuzTCJlpnbHfWUSpnOC4YJTlKFVh2guMomnaTteUOUejgWR7drU2unj9LpzPE9hKWlsckm+d466Hv4uDRt3L901fgjlYCYFbwJkYYtzDPRQROgmKSPpQKQTHzZWWGys92MYJfwwhVgyr3LkAh6fIj1Yd7y+NPldu0qARUVncRdlE8dYyQi+uwXs5/q9y+DuwpjykUYxL7sKlcAIEnC/x5joo0ZYRBIMZgHVI8vP8QR6MHygPnoFKo78NZ3GDhcI+jeQN21WgOZjgfh6wmOYOtC3zx3EkGcU78wAn2LLbo9Wbodvs88+nP0E9THn7jo7Tm6njDgoXFFrW2Q1cp6sD22HAU1urQsCpmNbkteW4qaccxDPpjer0BfhASRXU8D4ZDizAMiaKoZO6x8Dwb21HowiLNcrwsx7ItijzHKvP3nmd8MMOaY5EksWkb5jhEUY0oimg0mxRpzuLiIvW6acM1HA7JDx/Gn5tC3Xsf9f378DyfcHGBIk7ozkwbJXDxBVhZAwYQ7TL3fnDzZqLuia/hI9f69Ea36DfYWb3x7yKDUYbONLFt03f6+GHEnj0z+K7NcLiFUgmO55nSYK+Otm2sIiBNY9J0hE4TlM5RymRlknSLtaUNFndr9u0/yLQfcvfuRZKZjK2lC2w/t8Xa+YI/OnkWz7vE9ev/fyAVkXy51BFIcY8E20T4BHwjq7UIttQZbFAJqwTxhLE4wAi8EIZKJkBWZ4ERT5fb16kqDgXAMwlZlv19jAUhXALCRCS4ALEyhBdgm8oFaVPRi62Xn4PyeLvL46+zU5W40FYcax+izsHyZjimAMft4ixm7PXnaB/LuHezxpnNJs8tBzy/kvDEF06yee4U0cwCOu1x4dwpNro9rn72Caz6DO9+/7fTai/wuSc/w4MnTrC4N+LKtmJh2kx3ZRO63Zz9oYWP2ilzyLTJFozHMBhoup2Yra0B3W7MjFvD8wzyME0TwtBnZqYByiDjanWF7ymGI+j3FbnjYFkW49EYx7FptdsMBwPDL+g6KEz3H8dx8X1jZfi+b9h3Oh32HzjA9vY2QRCwtrqKHfkox2Lu6F1Mz8+SJjlWEZJlKXFR0KhFFHWbzZVnmbnvjTj7TrDy5J/dUgkc++p38Ms/9ZNsr63cZIu7gbMv/VoDSUK+ZdGxbTy3YNAbYzVCPC+k2XLQWnF9aY3la5fR2iZzLZzAp9YMaEQNPEujioQiS9haHzAaDHCLGqHlocYpe6dmuHr1KqdP9+h2gX5KvPoJXgnafOcogV1U8FqpEWhSlQRPU6XKFjDCOODGnoNgBHmNKli4jhHMqYlt5oB9mLSilP9O0onVMVaAVZ5rA4MZmGQrnkT4iVXgYoS6yY18gJqKZViCfBGVm2JRZUeEayCY2OdyeQ0t2JvCjGpNTKREJal7Uf5dBLs0/nyNVlon6GbEFzZYeeIUafwCUWue/QeOoHsJz/71E2yePc8wzviG73kXWysZf/Kffo63vPvdLByuc3Ed9s/CpW3YGuZsbA45e
```

Comment:
I would also explain why we want to use `nn.Sequential` instead of `Compose` (BC and because `Compose` is simply a `Sequential`)